modules/integration/tests-ui/src/test/resources/testng-server-mgt.xml

```diff
@@ -24,6 +24,7 @@
             <!--<class name="org.wso2.carbon.am.integration.ui.APIMANAGER2957EpochTimeTestCase"/>
             <class name="org.wso2.carbon.am.integration.ui.APIMANAGER3017ExternalStoreLifeCycleTestCase"/>
             <class name="org.wso2.carbon.am.integration.ui.APIMANAGER3149SSOLogin"/>-->
+            <!--<class name="org.wso2.carbon.am.integration.ui.APIResourceBodyParameterRemovalTestCase"/>-->
```

Comment:
In that case we need to skip the tests from local build and enable tests execution on Jenkins..We are woking on Jenkins part. 

To skip the test from regular build you can follow. 

Add the following property to surefire configuration

<skipTests>${skipUiTests}</skipTests>

Then add following property to ui module pom.

 <properties>

        <skipUiTests>true</skipUiTests>

    </properties>

To run the UI tests, you need to use following command.

mvn install -DskipUiTests=false